BastilleBSD/bastille
# network: Support adding/removing multiple interfaces in jails

This PR adds a subcommand called "network"

It allows you to add and remove interfaces to each jail, by adding/removing a network block to the jail.conf file.
It also makes some changes to clone and create, as the way bastille numbers it's interfaces is by counting the jails. This PR now counts them by grepping each jail.conf file for "epair" and "bastille" as that is what gets created when an interface is created.

To test -
Create a jail
Make sure everything functions as usual
Add an IF (bridged or otherwise) using the network subcommand
Add another IF i you prefer (only one entry per interface is allowed)
Restart to confirm interfaces function properly
Clone that jail
Restart the cloned jail
Confirm jail.conf and /etc/rc.conf are updated as they should be when cloning a jail

Note: cloned jails will set the default interface to the IP specified, and all other interfaces to SYNCDHCP. This can cause jails to take 10-20 seconds to fully start.

Do all the above with bridged vnet, vnet, and classic jails.
